src/chocolatey/infrastructure.app/nuget/NugetPush.cs

```diff
@@ -28,7 +28,7 @@ namespace chocolatey.infrastructure.app.nuget
 
     public class NugetPush
     {
-        public static void PushPackage(ChocolateyConfiguration config, string nupkgFilePath, ILogger nugetLogger, string nupkgFileName, IFileSystem filesystem)
+        public static void PushPackage(ChocolateyConfiguration config, string nupkgFilePath, ILogger nugetLogger, string nupkgFileName, IFileSystem filesystem, ChocolateySourceCacheContext cacheContext)

```

Comment:
As with the above comment, we need to add this as an overload instead of changing the existing method.

Reply:
And this!